packages/utils/src/hl7v2-notifications/send-hl7-message-to-mllp-server.ts

```diff
@@ -57,6 +57,7 @@ DG1|3|I10|I10.9^Essential (primary) hypertension^I10|Essential (primary) hyperte
 DG1|4|I10|E78.5^Dyslipidemia^I10|Dyslipidemia
 DG1|5|I10|E03.9^Hypothyroidism, unspecified^I10|Hypothyroidism, unspecified
 `;
+//ZFA|1|something|something|123 Main Street^Unit 1^Boston^MA^02108^USA
```

Comment:
delete / if leaving this for convenience leave it with a note about what context to use it in

Reply:
yeah oh i deleted my slash - i just meant if u wanna leave it in, leave a note about what its for.